src/handle_sign_offchain_message.h

```diff
@@ -1,10 +1,9 @@
+#pragma once
+
 #include "os.h"
 #include "cx.h"
 #include "globals.h"
 
-#ifndef _SIGN_OFFCHAIN_MESSAGE_H_
-#define _SIGN_OFFCHAIN_MESSAGE_H_
-
 void handle_sign_offchain_message(volatile unsigned int *flags, volatile unsigned int *tx);
 
-#endif
+void start_sign_offchain_message_ui(bool is_ascii, size_t num_summary_steps);
```

Comment:
Same thing, I find it weird not to have a dedicated ui header :thinking:

Reply:
src/ui/ui_api.h with all UI calls